code/clue.js

```diff
@@ -48,12 +278,59 @@ const pickKiller = () => {
 
   // This will change the background color of the card to the one connected to the chosen killer and show the full name of the killer. Feel free to add more things to show about the killer.
   document.getElementById('killerCard').style.background = mystery.killer.color
-  document.getElementById(
-    'killerName'
-  ).innerHTML = `${mystery.killer.firstName} ${mystery.killer.lastName}`
+  document.getElementById('killerImage').src = mystery.killer.image
+  document.getElementById('killerName').innerHTML = `${mystery.killer.firstName} ${mystery.killer.lastName}`
+  document.getElementById('killerAge').innerHTML = ` Age:${mystery.killer.age}`
+  document.getElementById('killerOccupation').innerHTML = `Occupation: ${mystery.killer.occupation}`
+  let favWeapon = weapons.find(weapon => weapon.id === mystery.killer.favouriteWeapon)
```

Comment:
Good job with `.find()` method!
Nonetheless, I don't see any place where you would overwrite value of `favWeapon` variable, so instead of declaring it with `let`, do it with `const`